main.go

```diff
@@ -83,10 +94,12 @@ func main() {
 	var out string
 
 	switch {
-	case args["markdown"].(bool):
-		out, err = markdown.Print(document, printSettings)
-	case args["md"].(bool):
-		out, err = markdown.Print(document, printSettings)
+	case args["markdown"].(bool), args["md"].(bool):
+		if args["--markdown-content"].(bool) {
+			out, err = content.Print(document, printSettings)
```

Comment:
Please use aliases `markdown_document` and  `markdown_table` in your package imports and re-use them here so that it reads `markdown_document.Print(...)`, etc..

Reply:
+1